.github/workflows/build.yaml

```diff
@@ -40,7 +57,7 @@ jobs:
       # in the auditwheel repair command, the '{package}' placeholder can't be used
       # Also rewrite the name in the pyproject.toml file.
       # TODO: Eventually this needs to be done more generically.
-      cibw-before-build: "cp {package}/.github/build-scripts/patch-wheel.sh /patch-wheel.sh && sed -i 's/name = \"ucx-py\"/name = \"ucx-py-cu11\"/g'  {package}/pyproject.toml"
+      cibw-before-build: 'cp {package}/.github/build-scripts/patch-wheel.sh /patch-wheel.sh && sed -i ''s/name = "ucx-py"/name = "ucx-py-cu11"/g''  {package}/pyproject.toml'
```

Comment:
Sorry for my ignorance of the yaml structure for GH, but is this duplicated single-quotes correct? Specifically around the sed expression. I would normally write it as:

sed -i "s/name = \"ucx-py\"/name = \"ucx-py-cu11\"/g"  {package}/pyproject.toml

Reply:
This wasn't actually a change I made manually. VS Code did this automatically when I saved the file.

After some Googling, I found the article below which talks about escaping the `'` character by doubling it within a larger single-quoted string:

- https://docs.octoprint.org/en/master/configuration/yaml.html#:~:text=In%20single%20quoted%20strings%20the,just%20indent%20the%20following%20lines

Ultimately, I'm not sure if the outer quotes here are necessary at all, since in YAML the following string values are equivalent:

```yaml
somekey: a value
anotherkey: "a value"
```

Probably not worth dwelling on too much unless you have a strong preference one way or the other.

Reply:
No, I don't have a preference, just thought it was unusual and not sure if that was even correct. If it works I don't mind keeping it as is.